packages/core/src/ceramic.ts

```diff
@@ -49,6 +49,7 @@ import * as path from 'path'
 import { buildIndexing } from './indexing/build-indexing.js'
 import type { DatabaseIndexApi } from './indexing/database-index-api.js'
 import { LocalIndexApi } from './indexing/local-index-api.js'
+import * as net from 'net'
```

Comment:
What is this? I don't see it being used anywhere.

Reply:
gah, must have been done my by IDE, I'll remove